documentation/src/docs/deployment/run-local.md

```diff
@@ -71,68 +72,59 @@ wget https://raw.githubusercontent.com/pravega/pravega/master/docker/compose/doc
+        singlenode.trustStoreJKS=../config/standalone.truststore.jks
 
-## Running standalone with encryption and authentication enabled
+     ```
 
-The configurations, `singlenode.enableTls` and `singlenode.enableauth` can be used to enable encryption and authentication respectively.(By default both these settings are disabled).
+2. Ensure that the server's certificate is trusted. To ensure the server's certificate is trusted, import it into the JVM's truststore. The server certificate used for the Pravega standalone mode server must be trusted on the JVM that runs the server. A server certificate can be rendered trusted via either Chain of Trust or via Direct Trust.
 
-In case `enableTls` is set to true, the default certificates provided in the `conf` directory are used for setting up TLS. These can be overridden by specifying in the properties file.
+    **Chain of Trust:** A chain of trust, which is the standard SSL/TLS certificate trust model, is established by verifying that the certificate is issued and signed by a trusted CA. If you are using a certificate issued by a CA as the server certificate, ensure that the CA's certificate is in the JVM's truststore.
 
-1. Ensure that the server's certificate is trusted. If you run `/gradlew startStandalone` without it, you'll encounter the following error:
+    **Direct Trust:** This type of trust is established by adding the server's certificate to the truststore. It is a non-standard way of establishing trust when using self-signed certificates. If you using self-signed certificates such as those provided by Pravega for standalone mode, ensure that the server's certificate is in the JVM's truststore.
 
-```java
-Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
-        at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:397)
-        at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:302)
-        at sun.security.validator.Validator.validate(Validator.java:260)
-        at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManager
-```
-2. To ensure the server's certificate is trusted, import it into the JVM's truststore. The following command sequence is used (in Linux) with the provided certificate file "cert.pem":
+The following command sequence is used (in Linux) with the provided certificate file `cert.pem` into the JVM's system truststore.
+
+   - `cd /path/to/pravega/config`
+   - Convert the `cert.pem` file to `DER` format: `openssl x509 -in cert.pem -inform pem -out cert.der  -outform der`
+   - Import the certificate into the local JVM's trust store:
+    `sudo keytool -importcert -alias local-CA -keystore /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/security/cacerts  -file cert.der` (using the default password `changeit`)
```

Comment:
please generalize the path to JRE here, as it can vary from one machine to another. Something like `/path/to/jre/lib/security/cacerts` works better here.